easybuild/easyconfigs/s/snap/snap-2013-11-29-GCC-6.4.0-2.28.eb

```diff
@@ -0,0 +1,40 @@
+#
+# Author: Fenglai Liu
+# fenglai@accre.vanderbilt.edu
+# Vanderbilt University
+#
+easyblock = 'MakeCp'
+
+name = 'snap'
+version = '2013-11-29'
+
+homepage = 'http://korflab.ucdavis.edu/software.html'
+description = """SNAP is a general purpose gene finding program suitable for both eukaryotic and prokaryotic genomes. 
+SNAP is an acroynm for Semi-HMM-based Nucleic Acid Parser."""
+
+toolchain = {'name': 'GCC', 'version': '6.4.0-2.28'}
+
+source_urls = ['http://korflab.ucdavis.edu/Software']
+sources = [SOURCE_TAR_GZ]
+patches = ['snap_makefile_correction.patch']
+checksums = [
+    'e2a236392d718376356fa743aa49a987aeacd660c6979cee67121e23aeffc66a',  # snap-2013-11-29.tar.gz
+    '02ea6dc6006ca5a5173dd2fc6a01b679a0910a68e0b51d00b7b9f732e4f79dcc',  # snap_makefile_correction.patch
+]
+
+files_to_copy = [
+    (["cds-trainer.pl", "hmm-assembler.pl", "noncoding-trainer.pl", "patch-hmm.pl", "zff2gff3.pl",
+      "exonpairs", "fathom", "forge", "hmm-info", "snap", "Zoe/zoe-loop"], "bin"),
+    "DNA",
+    "HMM",
+    "Zoe"
+]
+
+sanity_check_paths = {
+    'files': ["bin/snap", "bin/forge", "bin/zoe-loop"],
+    'dirs': [""],
+}
+
+modextravars = {'ZOE': '%(installdir)s/Zoe'}
```

Comment:
Also check for this location in `sanity_check_paths`?

Reply:
Zoe is a source dir. So it's already contained in the source folder. However, it seems the include files is needed by the other programs so they use the "ZOE" env variable to locate it.

Reply:
because it is always there, I think we do not need to put it into "sanity_check_paths".

Reply:
Oh, I forget this is the "Make then CP".... so the checking is still meaningful. I added the check for "Zoe" folder in the sanity_check_paths.